sale_commission_geo_assign/models/res_partner.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import models, fields, api
+
+
+class Partner(models.Model):
+    _inherit = 'res.partner'
+
+    country_ids = fields.Many2many(
```

Comment:
Maybe this field name is a bit too generic, something like `agent_country_ids` or similar would be better I think.
Same for other fields